modules/microshift-uninstalling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstalling-lvms-csi-snapshot_{context}"]
+== Uninstalling the CSI driver and CSI snapshot implementation
+
+To uninstall the installed CSI driver and CSI snapshot implementation, use the following procedure.
+
+.Procedure
+
+. To uninstall the CSI driver implementation, run all the following commands:
+* To delete the `vg-manager`, run the following command:
++
+[source,terminal]
+----
+$ oc delete -n openshift-storage daemonset.apps/vg-manager
+----
++
+.Example output
+[source,terminal]
+----
+daemonset.apps "vg-manager" deleted
+----
+* To delete the `lvms-operator`, run the following command:
++
+[source,terminal]
+----
+$ oc delete -n openshift-storage deployment.apps/lvms-operator
+----
++
+.Example output
+[source,terminal]
+----
+deployment.apps "lvms-operator" deleted
+----
+* To delete the `topolvm-provisioner`, run the following command:
++
+[source,terminal]
+----
+$ oc delete storageclasses.storage.k8s.io/topolvm-provisioner
+----
++
+.Example output
+[source,terminal]
+----
+storageclass.storage.k8s.io "topolvm-provisioner" deleted
+----
+
+. To uninstall the CSI snapshot implementation, run the following command:
```

Comment:
```suggestion
. Disable the CSI snapshot controller by running the following command:
```

Reply:
https://github.com/openshift/openshift-docs/pull/80823#discussion_r1745334831 same comment for not using disable and retaining "CSI driver and CSI snapshot implementation" through the docs

Reply:
retaining "uninstall" as per discussion